app/settings/account/-components/default-region/component.ts

```diff
@@ -12,15 +14,32 @@ import Toast from 'ember-toastr/services/toast';
 import RegionModel from 'ember-osf-web/models/region';
 import User from 'ember-osf-web/models/user';
 import CurrentUser from 'ember-osf-web/services/current-user';
+import buildChangeset from 'ember-osf-web/utils/build-changeset';
+import captureException from 'ember-osf-web/utils/capture-exception';
 
+interface RegionValidation {
+    defaultRegion: RegionModel;
+}
+
+const regionValidation: ValidationObject<RegionValidation> = {
+    defaultRegion: [
+        validatePresence({
+            presence: true,
+            ignoreBlank: true,
+            allowBlank: false,
```

Comment:
haven't gone through the validatePresence API yet but don't these two contradict?

Reply:
`ignoreBlank: true` will ignore empty and whitespace. I think allowBlank and ignoreBlank isn't useful for the default region though, since it's a power-selector, and shouldn't have any empty/blank inputs